Sources/TuistGenerator/Generator.swift

```diff
@@ -2,7 +2,7 @@ import Basic
 import TuistCore
 
 /// A component responsible for generating Xcode projects & workspaces
-public protocol Generating {
```

Comment:
I've made these as internal to avoid conflicts with the `Generator` I've temporarily added to `TuistKit` to help ease the migration to `TuistGenerator`